src/meta/src/hummock/manager/mod.rs

```diff
@@ -1430,13 +1432,18 @@ impl HummockManager {
                     VarTransactionWrapper,
                     VarTransaction::new(&mut versioning.version_stats,)
                 );
-                if let Some(table_stats_change) = &table_stats_change {
-                    add_prost_table_stats_map(&mut version_stats.table_stats, table_stats_change);
-                }
 
                 // apply version delta before we persist this change. If it causes panic we can
                 // recover to a correct state after restarting meta-node.
                 current_version.apply_version_delta(&version_delta);
+                if purge_prost_table_stats(&mut version_stats.table_stats, &current_version) {
```

Comment:
IMO, This metric is just an approximation, we don't need to update it every time we report the compact task, we can reduce the overhead by reporting it less frequently, what do you think?

Reply:
This PR only report the table which has update